parquet-common/pom.xml

```diff
@@ -67,6 +67,7 @@
       <artifactId>audience-annotations</artifactId>
       <version>0.12.0</version>
     </dependency>
+
```

Comment:
nit: Please undo formatting changes in this file.

Reply:
fixed